Bottom: f4de10c634e3bb5c3b719b7514aa69de6bf1fdef
Top:    d3e49d24e4878271eff300f7f4a96674069343ac
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 15:22:31 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index f4688ac..1f355ff 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2685,7 +2685,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
